hotel-bookings-api/FrontEnd/Hotel/src/Bookings.js

```diff
@@ -0,0 +1,158 @@
+    setEmail(event.target.value);
+  };
+  const handleRoomIdChange = (event) => {
+    setRoomId(event.target.value);
+  };
+
+  const handlecheckInDateChange = (event) => {
+    setCheckInDate(event.target.value);
+  };
+  const handlecheckOutDateChange = (event) => {
+    setCheckOutDate(event.target.value);
+  };
+
+  const newMessage = {
+    title,
+    firstName,
+    surname,
+    email,
+    roomId,
+    checkInDate,
+    checkOutDate,
+  };
+
+  fetch("https://hotel-bookings-server.onrender.com/bookings", {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify(newMessage),
+  })
+    .then((res) => res.json())
+    .then((data) => {
+      props(data);
+      setTitle("");
+      setFirstName("");
+      setSurname("");
+      setEmail("");
+      setCheckInDate("");
+      setCheckOutDate("");
+    });
+
+  return (
+    <form className="form">
+      <div>
+        <label className="label">
+          Title
+          <input
+            className="input"
+            type="text"
+            value={title}
+            onChange={handletitleChange}
+            required
+          />
+        </label>
+      </div>
+      <div>
+        <label className="label">
+          First Name:
+          <input
+            className="input"
+            type="text"
+            value={firstName}
+            onChange={handleFirstNameChange}
+            required
+          />
+        </label>
+      </div>
+      <div>
+        <label className="label">
+          Surname:
+          <input
+            className="input"
+            type="text"
+            value={surname}
+            onChange={handlesurNameChange}
+            required
+          />
+        </label>
+      </div>
+      <div>
+        <label className="label">
+          Email:
+          <input
+            className="input"
+            type="email"
+            value={email}
+            onChange={handleEmailChange}
+            required
+          />
+        </label>
+      </div>
+      <div>
+        <label className="label">
+          Room Id:
+          <input
+            className="input"
+            type="number"
+            value={roomId}
+            onChange={handleRoomIdChange}
+            required
+          />
+        </label>
+      </div>
+      <div>
+        <label className="label">
+          Check-In-Date:
+          <input
+            className="input"
+            type="date"
+            value={checkInDate}
+            onChange={handlecheckInDateChange}
+            required
+          />
+        </label>
+      </div>
+      <div>
+        <label className="label">
+          Check-Out-Date:
+          <input
+            className="input"
+            type="date"
+            value={checkOutDate}
+            onChange={handlecheckOutDateChange}
+            required
+          />
+        </label>
+      </div>
+      <div>
+        <button type="submit">Submit</button>
```

Comment:
the submit button will need a onClick event to trigger the onSubmitNewMessage function here.

```
<button type="submit" onClick={onSubmitNewMessage}>Submit</button>
```